innovation.game.php

```diff
@@ -11871,6 +11871,8 @@ function stPlayerInvolvedTurn() {
                 
             // id 174, Artifacts age 6: Marcha Real
             case "174N1":
+                self::setAuxiliaryValue(-1);
+                self::setAuxiliaryValue(-1);
```

Comment:
why is this done twice?

Reply:
Thanks for catching this bug! I meant to use setAuxiliaryValue2 the second time.